core/src/main/java/org/wso2/testgrid/core/command/CommandHandler.java

```diff
@@ -56,6 +56,8 @@
                                      impl = GenerateTestPlanCommand.class),
                          @SubCommand(name = "run-testplan",
                                      impl = RunTestPlanCommand.class),
+                         @SubCommand(name = "resolve-statuses",
+                                     impl = ResolveStatusesCommand.class),
```

Comment:
How about **`finalize-run-testplan`** instead?

Rationale: Whenever an exception occurred in the middle of run-testplan pipeline stage, then we need to finalize that partial test-run. Hence, if an exception occurred, then we need to run _finalize-run-testplan_.

Reply:
Created #602 to track this.